oteapi/models/parserconfig.py

```diff
@@ -8,5 +8,13 @@
 class ParserConfig(GenericConfig):
     """Parser Strategy Data Configuration."""
 
-    parserType: str = Field(..., description="Type of registered parser strategy.")
-    entity: AnyHttpUrl = Field(..., description="IRI to the entity or collection.")
+    parserType: str = Field(
+        ...,
+        description="Type of registered parser strategy.",
+        IRI="http://purl.org/dc/terms/type",
+    )  # type: ignore
+    entity: AnyHttpUrl = Field(
```

Comment:
Entity is a difficult name. It can mean anything. I am not found of freely mixing different vocabularies, but if we really want to identify this with `schema:url`, then I think the field should be named `url`.